packages/vm/src/cache.rs

```diff
@@ -108,6 +111,44 @@ where
         }
     }
 
+    /// Pins a Wasm that was previously stored via save_wasm.
+    /// This stores an already saved wasm into the pinned memory cache.
+    ///
+    /// The module is lookup first in the memory cache, and then in the file system cache.
+    /// If not found, the code is loaded from the file system, compiled, and stored into the
+    /// pinned cache.
+    /// If the given ID is not found, or the content does not match the hash (=ID), an error is returned.
+    pub fn pin_wasm(&mut self, checksum: &Checksum) -> VmResult<()> {
```

Comment:
Let's just call this `pin`/`unpin`. The `_wasm` is not really adding value here. Also we do not really pin the Wasm blob, but a compiled `Module` identified by the Wasm's checksum.